generate_case_list/case_list_conf.txt

```diff
@@ -4,17 +4,17 @@ cases_cna.txt	data_CNA.txt	<CANCER_STUDY>_cna	all_cases_with_cna_data	<CANCER_ST
+cases_complete.txt	data_RNA_Seq_expression_median.txt&data_CNA.txt&data_mutations_extended.txt	<CANCER_STUDY>_3way_complete	all_cases_with_mutation_and_cna_and_mrna_data	<CANCER_STUDY>	Complete samples	 Samples with mutation, CNA and expression data (<NUM_CASES> samples) 
+cases_complete.txt	data_mRNA_median_Zscores.txt&data_CNA.txt&data_mutations_extended.txt	<CANCER_STUDY>_3way_complete	all_cases_with_mutation_and_cna_and_mrna_data	<CANCER_STUDY>	Complete samples	 Samples with mutation, CNA and expression data (<NUM_CASES> samples) 
+cases_complete.txt	data_expression_Zscores.txt&data_CNA.txt&data_mutations_extended.txt	<CANCER_STUDY>_3way_complete	all_cases_with_mutation_and_cna_and_mrna_data	<CANCER_STUDY>	Complete samples	 Samples with mutation, CNA and expression data (<NUM_CASES> samples) 
 cases_log2CNA.txt	data_log2CNA.txt	<CANCER_STUDY>_log2CNA	all_cases_with_log2_cna_data	<CANCER_STUDY>	Samples with log2 copy-number data	Samples with log2 copy-number data (<NUM_CASES> samples)
 cases_methylation_hm27.txt	data_methylation_hm27.txt	<CANCER_STUDY>_methylation_hm27	all_cases_with_methylation_data	<CANCER_STUDY>	Samples with methylation data (HM27)	Samples with methylation (HM27) data (<NUM_CASES> samples)
 cases_methylation_hm450.txt	data_methylation_hm450.txt	<CANCER_STUDY>_methylation_hm450	all_cases_with_methylation_data	<CANCER_STUDY>	Samples with methylation data (HM450)	Samples with methylation (HM450) data (<NUM_CASES> samples)
-cases_methylation_promoters_hmEPIC.txt	data_methylation_promoters_hmEPIC.txt	<CANCER_STUDY>_methylation_promoters_hmEPIC	all_cases_with_methylation_data	<CANCER_STUDY>	Tumor samples with methylation data (Promoters, hmEPIC)	All samples with methylation (Promoters, hmEPIC) data (<NUM_CASES> samples)
-cases_methylation_genebodies_hmEPIC.txt	data_methylation_genebodies_hmEPIC.txt	<CANCER_STUDY>_methylation_genebodies_hmEPIC	all_cases_with_methylation_data	<CANCER_STUDY>	Tumor samples with methylation data (Gene Bodies, hmEPIC)	All samples with methylation (Gene Bodies, hmEPIC) data (<NUM_CASES> samples)
-cases_methylation_promoters_wgbs.txt	data_methylation_promoters_wgbs.txt	<CANCER_STUDY>_methylation_promoters_wgbs	all_cases_with_methylation_data	<CANCER_STUDY>	Tumor samples with methylation data (Promoters, WGBS)	All samples with methylation (Promoters, WGBS) data (<NUM_CASES> samples)
-cases_methylation_genebodies_wgbs.txt	data_methylation_genebodies_wgbs.txt	<CANCER_STUDY>_methylation_genebodies_wgbs	all_cases_with_methylation_data	<CANCER_STUDY>	Tumor samples with methylation data (Gene Bodies, WGBS)	All samples with methylation (Gene Bodies, WGBS) data (<NUM_CASES> samples)
+cases_methylation_promoters_hmEPIC.txt	data_methylation_promoters_hmEPIC.txt	<CANCER_STUDY>_methylation_promoters_hmEPIC	all_cases_with_methylation_data	<CANCER_STUDY>	 Tumor samples with methylation data (Promoters, hmEPIC) 	 All samples with methylation (Promoters, hmEPIC) data (<NUM_CASES> samples) 
```

Comment:
And here also?

Reply:
Same here. There were extra spaces that got removed. Nothing changed with the data.